doc/src/selections.rst

```diff
@@ -73,15 +97,34 @@ String properties
 Numeric properties
 ------------------
 
+Most of the numeric properties only apply to a single atom:
+
 - ``index``: gives the atomic index in the frame;
 - ``mass``: gives the atomic mass;
 - ``x``, ``y`` and ``z``: gives the atomic position  in cartesian coordinates;
 - ``vx``, ``vy`` and ``vz``: gives the atomic velocity in cartesian coordinates;
 - ``resid``: gives the atomic residue index. If an atom is not in a residue,
   this return -1;
 
-Supported functions in mathematical expressions are: ``sin``, ``cos``, ``tan``
-for the trigonometric functions; ``asin`` and ``acos`` inverse trigonometric
+But some properties apply to multiple atoms, and as such are only usable when
+selecting multiple atoms:
+
+- ``distance(i, j)``: gives the distance in Ångströms between atoms i and j,
+  accounting for periodic boundary conditions.
+- ``angle(i, j, k)``: gives the angle between atoms i, j and k in radians,
+  accounting for periodic boundary conditions. The atoms do not need to be
+  bonded together.
+- ``dihedral(i, j, k, m)``: gives the dihedral angle between atoms i, j, k and m
+  in radians, accounting for periodic boundary conditions. The atoms do not need
+  to be bonded together.
+- ``out_of_plane(i, j, k, m)``: gives the distance in Ångströms between the
```

Comment:
Maybe rename out_of_plane to improper to match the is_improper method.

Reply:
As I understand it, improper dihedral angles can be used in two different ways when computing potentials: either as dihedral angles (the angle between ij and the jkm plane), or as an out of plane distance (distance between j and the ikm plane). This is why the functions are named this way. 

```
    i
    |
    j
  /   \
k       m
```

Does it makes sense ? I am not really used to work with impropers.